### ISSUE TITLE
[buy] The BUY/PURCHASE page for a Seller listing should have the Asset details component

### ISSUE THREAD

**Issue by jabby09** [KlimaDAO/klimadao#2018]
See a BUY/PURCHASE page for a pool project listing, for example https://www.carbonmark.com/projects/VCS-191-2010/purchase/pools/bct

Note the collapsed by default Asset details component.
<img width="1097" alt="Screenshot 2023-12-12 at 9 26 08 AM" src="https://github.com/KlimaDAO/klimadao/assets/103535195/15d9f10d-0076-4b61-b2b5-0c9853e0da8c">

On the same project, if I choose to BUY from a seller listing, for example https://www.carbonmark.com/projects/VCS-191-2010/purchase/0xa591682126238e780ae64b5c976feaadb28179730ece22613c15672c9cd004b8

The Asset details component is missing.
<img width="1103" alt="Screenshot 2023-12-12 at 9 27 03 AM" src="https://github.com/KlimaDAO/klimadao/assets/103535195/757e2566-37df-4ae0-8e97-f7fc8718241f">

### Requirements

- Add the Asset details component to the BUY page of a Seller listing
<img width="366" alt="Screenshot 2023-12-12 at 9 27 56 AM" src="https://github.com/KlimaDAO/klimadao/assets/103535195/65c0b1d1-6269-46c7-9fb1-cf3041a4e9b2">

**Notes**
- `Token you will receive` I believe is not coming in through the api and requires a subgraph query (to be confirmed with atmos)
- `Available to purchase` will be the total quantity of listing

**Comment by jabby09** [comment]
@biwano if you run in to issues here trying to find the data you need in the api, talk to @wowselim and @Atmosfearful as I understand they are creating a new listings endpoint on the api for seller data.

**Comment by biwano** [comment]
> @biwano if you run in to issues here trying to find the data you need in the api, talk to @wowselim and @Atmosfearful as I understand they are creating a new listings endpoint on the api for seller data.

Hi @wowselim @Atmosfearful,

As per @jabby09 advice, I am seeking help. The current `/projects/:project_id` endpoint returns the project with listings. But the listing objects contain information about the project token being sold (the credit if I understood properly), but does not contain information about the token type (eg. C3T/TCO2)
I dived a little bit in the marketplace subgraph and I did not find the information either.

I think I can deduce it from the project pool prices but I dont think it would be a lasting solution. (what if we have multiple pools/no pools etc...)
